### PR TITLE
Feature/arangodb 3.11

### PR DESCRIPTION
**Pull request by KVS85**
- Add 3.10 and 3.11 scripts for packages and remove 3.7
- Add 3.11 docker containers and remove 3.7
- Remove 3.7-related build images

Jenkins PR: https://jenkins.arangodb.biz/view/PR/job/arangodb-matrix-pr/29347/

### CODE REVIEW

File: debian/3.11/community/copyright [hunk 1]
```diff
@@ -0,0 +1,17 @@
+This is the Debian GNU/Linux prepackaged version of arangodb.
+It was packaged by Frank Celler <f.celler@arangodb.org>
+from sources obtained from http://github.com/arangodb/ArangoDB.
+
+Copyright:
+
+Copyright (C) 2012-2013 triAGENS GmbH
+              2014-2021 ArangoDB GmbH
```
Comment (dothebart) [inline debian/3.11/community/copyright:8]: extend license

File: debian/3.11/enterprise/control [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+Architecture: @ARCHITECTURE@
+Depends: ${misc:Depends}
+Conflicts: arangodb3e-client, arangodb3, arangodb3-client
+Description: a multi-model NoSQL database
+ A distributed free and open-source database with a flexible data model for documents,
+ graphs, and key-values. Build high performance applications using a convenient
+ SQL-like query language or JavaScript extensions.
+ .
+ Copyright: 2012-2013 by triAGENS GmbH
+ Copyright: 2014-2021 by ArangoDB GmbH
```
Comment (dothebart) [inline debian/3.11/enterprise/control:17]: extend license

File: dmg/3.11/ArangoDB3e-CLI.app/Contents/Info.plist [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  <string>acli</string>
+
+  <key>CFBundleVersion</key>
+  <string>@VERSION@</string>
+
+  <key>NSAppleScriptEnabled</key>
+  <string>NO</string>
+
+  <key>NSHumanReadableCopyright</key>
+  <string>Copyright © 2014-2021 ArangoDB GmbH, Cologne, Germany. All rights reserved.</string>
```
Comment (dothebart) [inline dmg/3.11/ArangoDB3e-CLI.app/Contents/Info.plist:42]: extend license

File: dmg/3.11/ArangoDB3-CLI.app/Contents/Info.plist [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+  <string>acli</string>
+
+  <key>CFBundleVersion</key>
+  <string>@VERSION</string>
+
+  <key>NSAppleScriptEnabled</key>
+  <string>NO</string>
+
+  <key>NSHumanReadableCopyright</key>
+  <string>Copyright © 2014-2021 ArangoDB GmbH, Cologne, Germany. All rights reserved.</string>
```
Comment (dothebart) [inline dmg/3.11/ArangoDB3-CLI.app/Contents/Info.plist:42]: extend license

File: rpm/3.11/arangodb3.spec.in [hunk 5]
```diff
@@ -0,0 +1,329 @@
+#
+# spec file for package arangodb3
+#
+# Copyright (c) 2014-2021 info@arangodb.org
```
Comment (dothebart) [inline rpm/3.11/arangodb3.spec.in:4]: please extend the license

File: rpm/3.11/arangodb3e.spec.in [hunk 6]
```diff
@@ -0,0 +1,331 @@
+#
+# spec file for package arangodb3
+#
+# Copyright (c) 2014-2021 info@arangodb.org
```
Comment (dothebart) [inline rpm/3.11/arangodb3e.spec.in:4]: please extend the license

File: snippets/3.11/download-index.html [hunk 7]
```diff
@@ -14,7 +14,7 @@
         <meta charset="utf-8"/>
         <meta name="robots" content="noindex" />
         <title>
-            Index of /arangodb37
+            Index of /arangodb310
```
Comment (dothebart) [inline snippets/3.11/download-index.html:17]: 311 ?

File: snippets/3.11/download-index.html [hunk 8]
```diff
@@ -62,7 +62,7 @@
         </style>
     </head>
     <body>
-      <h1>Index of /arangodb37</h1>
+      <h1>Index of /arangodb310</h1>
```
Comment (dothebart) [inline snippets/3.11/download-index.html:65]: 311?

**Review by dothebart — APPROVED** [review]
please adjust the copyright date ranges so everything looks fresh. 

snippets/3.11/download-index.html probably needs to print 311 directory?

**Review by CryptoNinjaGeek — APPROVED** [review]

**Pull request merged**